packages/extension-core/src/notifications/createNotification.ts

```diff
@@ -36,6 +36,17 @@ const getNotificationOptions = (
           `Failed transaction on ${networkName}.`,
         iconUrl: "/images/tx-nok.png",
       }
+    case "not_found":
+      return {
+        type: "basic",
+        title: "Transaction not found",
+        message:
+          error?.shortMessage ??
+          error?.reason ??
+          error?.message ??
+          `Transaction not found on ${networkName}.`,
```

Comment:
I'm not sure if this will be a meaningful message for a lot of users. How about: "We aren't able to determine the status of this transaction"

And since we know which error will have caused it, we don't need the `error?.shortMessage ??` etc in there.